### DISCUSSION TITLE
動畫中的「未來科技」，對未來的想像?

### DISCUSSION THREAD

**Discussion by 112121py** [1101-GS4719-C/Python-Community-GS4719-C#2145] [General]
人們常常在影視作品、漫畫小說中，盡情發揮對未來的想像跟創意。這邊以動畫做舉例，我列了幾部蠻好看的xd
### Cyberpunk: EdgeRunners 電馭判客(2077)
是一部描述在夜城生活的人們的科幻悲劇故事，人們可以透過植入「沙德威斯坦裝置」來強化自己的身軀，可以把自己改造，有機器班強大的能力，但缺點就是過度改造自己，容易「神機錯亂」，失去自我...

### 刀劍神域 (SAO)
SAO篇描述了一個在2022年發售的VR MMORPG遊戲，使用者戴上一個長得像安全帽xd的頭盔，就可以連線登入遊戲世界，但遊戲設計者在玩家登入後，說明這雖然是遊戲，但可不是鬧著玩的。只要你在遊戲中死亡，即會立刻在現實當中也死亡，除了通關沒有任何辦法。

愛麗絲篇則描述了一個特別的人工搖光技術，在一個他們創造的世界(Under World)裡，讓這些搖光(類似靈魂)自行生存，進而發展成一座有規模的城市，還有邪惡勢力。進而也探討了他們創造出來的存在──愛麗絲，他們做的人工智慧算不算是人類。但這篇篇幅蠻長的，世界觀也很龐大，探討了很多事情，有興趣可以自行去看看，沒有辦法太簡單的描述xd
~~至於要看動畫還是小說，自行斟酌www~~

### 小結
其他作品像尼爾: 人工智慧、可塑性記憶、deca-dence、Vivy -Fluorite Eye's Song-等等，都有描述一些對未來科技的想像，或是對AI的探討，都蠻值得一看，不知道大家對未來科技的想法、對其的想像或是希望有什麼技術出現，也可以推薦一些科幻作品~~

**Comment by seastar0687** [comment]
其實在上個世紀的動畫電影有許多對於這類的想像，最有名的例子就是大友克洋的⟪阿基拉⟫，隨然這部作品的重點並不是未來科技（其實主角的能力更像超能力XD），但可以看現今許多對於賽博龐克的許多想像在那時候就已經成熟了。而通常這類作品主要還是通過科幻的設定來探討角色的心境與變化跟冒險，而非探討科技本生，不過科幻的有趣之處就是比以奇幻更多了一分可能成真的醍醐味，所以其中科技帶來各種好處與壞處都不禁讓人更加地會去反思呢。

**Reply by 112121py** [reply]
感謝同學的推薦，有機會會去看一下⟪阿基拉⟫。
但我也蠻認同類似的科幻作品通常是在探討角色的成長及變化，但也不免地可以讓我們反思科技可能會對我們帶來的影響，有更多東西可以讓我們思考或發揮想像力xd
~~像是都2024年了為什麼SAO還沒被發明~~

**Comment by Chattenoire** [comment]
若你對未來科技有興趣，特別是像SAO這類的虛擬實境遊戲，那推薦你看一位法國哲學家寫的書《虛擬真實：我們的身體在或不在？》

書中對於人們關於虛擬空間的迷思/謬誤/幻想皆有批判，讀完後也許可以更實際的去期待未來的"虛擬世界"。
(當然，有興趣的話可以深入閱讀更多，像是Deleuze的著作或是其他科技哲學文章)

**Comment by Taoyi-Chiang** [comment]
《電馭叛客2077》我是玩遊戲的版本，故事真的寫得很好，玩到哭。
相較於對虛擬世界的嚮往，我覺得在遊戲中更強調的是如何對抗孤獨，或更準確的說是人與生俱來的寂寞感。
對比《魔獸世界》或和《電》同一個工作室出品的《巫師三》，奇幻文學更傾向於訴說一個「我可以是誰」的英雄主義故事，因此總是有數不盡的與邪惡對抗作為成為英雄（或者勇士）的過程。
相反的以賽博朋克為題材的遊戲，當然也應該包含《鬼線：東京》，似乎對於「我是誰」的討論有更多的偏好，這類題材會更著眼於探討自我存在的價值或意義。所以這兩款遊戲也都可以看到一個身體存在兩個靈魂，這種設定本身就是對自我的掌控提出了否定。
從這個角度來說，我覺得《刀劍神域》以及甚至幾乎所有的日本轉生動畫，可能都依然是在奇幻文學的框架之下，而沒有太多的賽博朋克要素。

**Comment by christaaah** [comment]
我覺得動漫真的是發揮想像力的很方便的媒介，對於高科技進行倫理上的探討也很引人深思。雖然可能有些很偏離目前的現實，但是未來的是誰說得準呢？很多以前不敢想像的東西現在就已經變成日常得必需品了，這些東西在未來也可以會變成稀鬆平常的存在也不一定。

**Comment by lai1008** [comment]
我覺得漫畫真的是很容易讓我們的想像力爆發，也讓我們對這些未來可能會出現的科技好好地想想背後的後果。雖然可能有些很偏離目前的現實，但是未來會如何發展誰知道呢？在20年前應該也沒有人會相信20年後的今天會有那麼強大的AI出現，很多以前不敢想像的東西現在就已經變成日常得必需品了，這些東西在未來也可以會變成稀鬆平常的存在也不一定。
